core/src/main/java/org/apache/accumulo/core/clientImpl/TableOperationsImpl.java

```diff
@@ -543,9 +543,11 @@ public void putSplits(String tableName, SortedMap<Text,TabletMergeability> split
         }, waitExecutor));
       }
 
+      record FateSplits(TFateId opid, List<SplitMergeability> splits) {
```

Comment:
This is declared in the method, the code was a using a Pair to pass two things back from a background computation done in the method.